docs/Parameters.rst

```diff
@@ -139,6 +139,12 @@ Core Parameters
 
       -  **Note**: internally, LightGBM uses ``gbdt`` mode for the first ``1 / learning_rate`` iterations
 
+-  ``data_sample_strategy`` :raw-html:`<a id="data_sample_strategy" title="Permalink to this parameter" href="#data_sample_strategy">&#x1F517;&#xFE0E;</a>`, default = ``bagging``, type = enum, options: ``bagging``, ``goss``
+
+   -  ``bagging``, Randomly Bagging Sampling
```

Comment:
I think we should add a note in this section of the docs that since the default for `bagging_fraction` is `1.0`, the default behavior of LightGBM is "use all observations in the training data at every iteration".

If I didn't understand LightGBM's internals, I might think that "default `data_sample_strategy=bagging`" means that bagging is the default behavior of LightGBM, when it actually is not.

Reply:
Thanks. What about adding that `only effective when bagging_freq > 0 and bagging_fraction < 1.0`?

Reply:
It should be `bagging is only effective when bagging_freq > 0 and bagging_fraction < 1.0`. Because `bagging_freq` and `bagging_fraction` have no effect on GOSS.

Reply:
Yep that works!

Reply:
Resolved via 4bdcdd5